resources/function_help/json/bearing

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "bearing",
+  "type": "function",
+  "groups": ["GeometryGroup"],
+  "description": "Returns the north-based bearing as the angle in radians measured clockwise on the ellipsoid from the vertical on point_a to point_b.",

```

Comment:
Degrees would be more useful:

```suggestion
  "description": "Returns the north-based bearing as the angle in degrees measured clockwise on the ellipsoid from North on point_a to point_b.",

```

Reply:
Thank you for the reviewing work. The idea was to make the bearing expression function behave like the azimuth expression function, which returns the angle in radians in the range [0, 2π[ (while the QgsDistanceArea::bearing function returns the value in the range ]-π, π]). The bearing expression function returns the same value of the azimuth expression function when the ellipsoid parameter is set to NONE (or, if the ellipsoid parameter is not provided, when the project ellipsoid in set to none/planimetric ).

I agree that the output in degrees would be more useful to most of the user. If it's not the problem the inconsistency with the azimuth expression function, I will change the output of the function to be in degree in the range [0-180[.

Reply:
My gut feeling is to go with degrees, even if it's slightly inconsistent with other functions

Reply:
Actually, ok... let's leave as radians